CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 2.7.0-rc.0 / 2019-01-18
+
+This release adds support for PromQL subqueries and disk size based retention. Please note that the disk size calculations currently only account for space occupied by blocks, and not the WAL (which could be of a significant size). You can set both time and disk size retention policies, with time based retention defaulting to 15d when unset. Further, the disk based retention is an experimental feature with no stability guarantees. 
```

Comment:
Details on specific features either belongs in the docs, or on the relevant line below.